### PR TITLE
chore: Fixed up CI and build for running

### PR DESCRIPTION
**Pull request by TheBeege**

### CODE REVIEW

**Comment by TheBeege** [comment]
Fixes #1

File: Dockerfile
Comment (catagris) [inline Dockerfile:1]: The Dockerfile uses two base images: gradle:jdk17-alpine and openjdk:17-jre-slim. Can we reduce this to one so it has a smaller image size and make it more efficient?

Comment (TheBeege) [reply Dockerfile:1]: Ah, no. This diff is saying that using the gradle and openjdk images is removed. Using only eclipse-temurin is added.

**Review by catagris — APPROVED** [review]
Ah, I can see that now. Misread the edit.

File: Dockerfile
Comment (catagris) [reply Dockerfile:1]: Awesome! Sorry for the misread.

**Pull request merged**